src/zoeker.js

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios'
+
+// 1. installeer en importeer axios
+// ---- zoek in de API documentatie het juiste endpoint
+// 2. asynchrone functie schrijven
+// 3. try - catch blok
+// 4. In het try blok gaan we ons GET-request maken met axios
+// 5. Resultaat loggen
+// 6. Probeer een console.log() met de naam van het land te maken
+// 7. Probeer een console.log() met de populatie van het land
+
+// 8. Maak een if/else statement voor valuta's per land
+
+// DINGEN OP PAGINA WEERGEVEN STAPPENPLAN
+// 1. 'anker'-element in het HTML-bestand plaatsen (met ID!)
+// 2. sla de referentie naar dit element op in het JS-bestand
+// 3. Gebruik .innerHTML om een stukje data op de pagina te laten zien
+
+const searchForm = document.getElementById("searchField");
+searchForm.addEventListener('submit',getInput);
+
+function getInput(e){
+    e.preventDefault()
+    // console.log("test")
+    const searchInput = document.getElementById("search");
+    getCountries(searchInput.value);
+    // console.log(searchInput.value);
+}
+
+async function getCountries(name) {
+    const error = document.getElementById("errorMessage");
+    error.innerHTML = "";
+    try {
+        const result = await axios.get(`https://restcountries.com/v2/name/${name}`)
+        console.log(result)
+        console.log(result.data[0].languages)
+        const searchBar = document.getElementById('country-info1');
+
+        if (result.data[0].languages.length === 1) {
```

Comment:
Het doet pijn aan m'n ogen om zoveel dubbele code te zien. Wat je had kunnen doen is een functie schrijven die een string teruggeeft voor zowel de taal als de munteenheden. Dan kun je de rest van alle HTML elementen laten zoals het is, en kun je de waarde voor de taal een munteenheden gewoon via die functie bepalen